hack/build/fake.go

```diff
@@ -1,7 +1,7 @@
 package fake
 
-// Commented out import ERROR: /home/runner/work/cluster-api-provider-hcloud/cluster-api-provider-hcloud/hack/build/BUILD.bazel:17:11: in go_library rule //hack/build:go_default_library: target '@com_github_tcnksm_ghr//:go_default_library' is not visible from target '//hack/build:go_default_library'. Check the visibility declaration of the former target if you think the dependency is legitimate
 // +build ignore
+
```

Comment:
This file is necessary to import ghr, which is used to release a new version to GitHub